CONTRIBUTING.md

```diff
@@ -558,17 +562,17 @@ arguments and return values.
 
 ### Code Review
 
-After you've submitted a pull request, you should expect to hear at least a comment
-within a couple of days.
-We may suggest some changes or improvements or alternatives.
+After you've submitted a pull request, you should expect to hear at least a
+comment within a couple of days. We may suggest some changes, improvements or
+alternative implementation details.
 
-Some things that will increase the chance that your pull request is accepted quickly:
+To increase the chances of getting your pull request accepted quickly, try to:
 
 * Write a good and detailed description of what the PR does.
-* Write tests for the code you wrote/modified.
-* Readable code is better than clever code (even with comments).
-* Write documentation for your code (docstrings) and leave comments explaining the
-  *reason* behind non-obvious things.
+* Write tests for the code you wrote/modified if needed.
+* Use readable code, as it is better than clever code (even with comments).
+* Write some documentation for your code (docstrings) and leave comments explaining
+  the *reason* behind non-obvious things.
 * Include an example of new features in the gallery or tutorials.
 * Follow the [PEP8](http://pep8.org) style guide for code and the
   [numpy guide](https://numpydoc.readthedocs.io/en/latest/format.html)
```

Comment:
**Numpy style guide** is cited by three times.
Line 318-319 in [Docstring](https://github.com/GenericMappingTools/pygmt/blob/revise-pr-review-process/CONTRIBUTING.md#docstrings):
> **All docstrings** should follow the [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard).

Line 558-559 in [Documentation](https://github.com/GenericMappingTools/pygmt/blob/revise-pr-review-process/CONTRIBUTING.md#documentation)
> **All docstrings** should follow the [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html).

```suggestion
  [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html)
```

---

The link in line 318-319 in [Docstring](https://github.com/GenericMappingTools/pygmt/blob/revise-pr-review-process/CONTRIBUTING.md#docstrings) is a little different: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

The other two use: https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
> I cannot add suggestions for the next line (579). "Documentation" should be "docstrings", right? I feel documentation contain tutorials, gallery, docstrings, and other texts.

Ok, changed documentation -> docstrings at 78d33a21dacb91ea798ecf0567aba0b5fc3f94f2.